packages/data-provider/src/utils/buildQuery.ts

```diff
@@ -0,0 +1,123 @@
+import { SQL, and, not, or, sql } from 'drizzle-orm';
+import * as _ from 'lodash-es';
+import { BadRequest } from './errors.js';
+import {
+	ArrayFilterValue,
+	CombinationKeys,
+	CombinationOperator,
+	FilterOperator,
+	GreaterThanFilter,
+	LesserThanFilter,
+	Operator,
+	isArrayFilter,
+	isCombination,
+	isFilter,
+} from '@overture-stack/sqon-builder';
```

Comment:
These should be sorted by import type and separated by an empty line.
Does Lyric include that prettier plugin that helps with this?

Reply:
Reorganized imports. somehow prettier didn't organized them earlier...
Lyric has prettier plugin `prettier-plugin-organize-imports` however this doesn't add an empty line. wondering what configuration does this.